src/inference/tests/unit/config_test.cpp

```diff
@@ -145,9 +145,7 @@ struct NotEmptyTestConfig : public ov::PluginConfig {
         if (!is_set_by_user(low_level_property)) {
             m_low_level_property.value = m_high_level_property.value;
         }
-#ifdef ENABLE_DEBUG_CAPS
         apply_config_options(device_name, test_config_path);
-#endif
```

Comment:
Are you going to expose config options as well?

Reply:
oh, right. it was my mistake..